src/pages/panel/views/simple.js

```diff
@@ -18,40 +18,14 @@ import {
   chevronRight,
 } from '/vendor/@whotracksme/ui/src/components/icons.js';
 
-import { toggles } from '../utils/rulesets.js';
+import { rulesetIds } from '../utils/rulesets.js';
+import sites from '../../../rule_resources/sites.json';
```

Comment:
not sure about this. `sites.json` is relatively big, so importing it will slow down rendering. It should rather be loaded asynchronously, or maybe ask background to make the check.

Reply:
Hmm.. as it is a "site", we can use here dynamic import, so if you think it can slow it down, we can check it out. However, I would do some basic time checks, as i think js files might be precompiled, so it might not have impact on load time.

Reply:
I made some basic performance tests and it seems that there is no difference between loading sites as static import or by dynamic import. The FCP (first content paint) of the panel page is on the same more or less 100 ms level. I also checked how downloading and compiling the file impacts the perf. Here you have some screenshots:

<img width="475" alt="Zrzut ekranu 2022-04-4 o 09 50 27" src="https://user-images.githubusercontent.com/1906677/161499704-d17556fb-4c9e-4691-8a1d-1c462a181dcc.png">
<img width="152" alt="Zrzut ekranu 2022-04-4 o 09 51 11" src="https://user-images.githubusercontent.com/1906677/161499710-3ce345e6-abd0-421f-adca-922f4f51b3ae.png">

As you can see, download and parse takes less than 7ms and compiling less than 2ms :) 

In my opinion, I would stay with static import, as it is simpler in form for using the data, and does not impact the performance.